internal/csi-addons/rbd/replicationcontrollerserver.go

```diff
@@ -325,61 +326,14 @@ func (rs *ReplicationServer) DisableVolumeReplication(ctx context.Context,
 	case librbd.MirrorImageDisabling:
 		return nil, status.Errorf(codes.Aborted, "%s is in disabling state", volumeID)
 	case librbd.MirrorImageEnabled:
-		return disableVolumeReplication(rbdVol, mirroringInfo, force)
+		return rbdutil.DisableVolumeReplication(rbdVol, mirroringInfo, force)
```

Comment:
`rbdutil` functions should not return a GRPC response. Only the packages under `internal/csi-addons/` should use the CSI-Addons GRPC structures/functions.

Maybe `DisableVolumeReplication()` can return an error instead, which can then be used to construct a `DisableVolumeReplicationResponse` reply here?